core/controllers/reader.py

```diff
@@ -335,25 +335,24 @@ def _require_playthrough_data_is_valid(self, playthrough_data):
         """
         playthrough_properties = [
             'exp_id', 'exp_version', 'issue_type', 'issue_customization_args',
-            'playthrough_actions']
+            'actions']
```

Comment:
Similar to my previous comment... but since this is already in we can worry about it later. No action needed for this PR.

Reply:
Done,